gcc/rust/rust-session-manager.cc

```diff
@@ -474,6 +498,26 @@ Session::enable_dump (std::string arg)
 void
 Session::parse_files (int num_files, const char **files)
 {
+  rust_assert (num_files > 0);
```

Comment:
Is this `assert` necessary? I am not quite sure.

Reply:
yeah this will cause an assertion if someone invokes the compiler directly without any inputs which is valid to produce an empty compilation unit. So we need to remove that

Reply:
> yeah this will cause an assertion if someone invokes the compiler directly without any inputs which is valid to produce an empty compilation unit. So we need to remove that

Hmm. I will provide a fallback path for this case.